bedrock/products/templates/products/monitor/cms/includes/macros.html

```diff
@@ -0,0 +1,37 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% macro picto_icon(image) -%}
+  {# TKTK: To do: this is SVG, input should only accept SVG, this is one size #}
```

Comment:
See my comment above about using a direct FileField instead of an Image for SVGs, as they don't need any of the rendition processing

Or, if we do stick with the code for a `ForeignKey` to `cms.BedrockImage`, we can just directly reference the image here (`image.file` or similar - can't remember off top of my head) and include it directly withotut the srcset helper or worryign about any renditions of the image.

We could also do some work to add validation to that `cms.BedrockImage` class to allow us to spec which file types are allowed on a use-by-use basis. Happy to chat about that option, too, if you think it's useful